app/models/pair_request.rb

```diff
@@ -2,17 +2,29 @@
 #
 # Table name: pair_requests
 #
-#  id          :bigint           not null, primary key
-#  duration    :float
-#  when        :datetime
-#  created_at  :datetime         not null
-#  updated_at  :datetime         not null
-#  acceptor_id :integer
-#  author_id   :integer
+#  id         :bigint           not null, primary key
+#  duration   :float            not null
+#  status     :integer          default(0), not null
+#  when       :datetime         not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  author_id  :integer          not null
+#  invitee_id :integer          not null
 #
 class PairRequest < ApplicationRecord
   belongs_to :author, class_name: "User"
-  belongs_to :acceptor, class_name: "User", optional: true
+  belongs_to :invitee, class_name: "User"
 
-  validates :when, presence: true
+  validates :when,
+            presence: true,
+            inclusion: { in: (Date.today..(Date.today + 1.month)) }
+  validates :duration, presence: true, numericality: { greater_than_or_equal_to: 900.0 }
```

Comment:
what is the duration value actually (hours?)

Reply:
The duration is 900 seconds wich is 15 minutes

Reply:
why are we doing seconds?? This should either be minutes (integer) or hours (float). There isn't a purpose to making a meeting 92 seconds. Personally I think it should be minutes and we store as an integer?